Godeps/Godeps.json

```diff
@@ -128,13 +128,18 @@
 		},
```

Comment:
@jojimt I think he means:

```sh
make godep-save
git add Godeps
git add vendor
git commit -m "updated Godeps"
```

Just so that it generates a separate commit, not a separate PR.  As an example, I just did this here:  https://github.com/contiv/objdb/pull/20